ActionLogger/src/pl/jtat/logger/model/EventData.java

```diff
@@ -0,0 +1,62 @@
+package pl.jtat.logger.model;
+
+import pl.jtat.logger.C;
+import pl.jtat.logger.util.DateFormatter;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+public class EventData {
+
+    private UUID eventId;
+    private EventType eventType;
+    private Date eventDate;
+    private Boolean eventResult;
+    private List<KeyValueData> additionalEventData;
```

Comment:
Nie można by wykorzystać dostępnego w Java typu Map?
Wtedy nie trzeba byłoby implementować własnego Pair

Reply:
Zostaje List<KeyValueData> do momentu kiedy zostanie określony zbiór dodatkowych atrybutów - żeby zobrazować o co mi chodzi: sala może mieć 2 atrybuty: nazwę i liczbę miejsc. Użycie Map<String,Integer> nie sprawdzi się w obu przypadkach, a Map<String,String> mija się z celem, żeby nie wrzucać Integera jako Stringa.

Reply:
To przechowuj w Map<String, Object>

Reply:
A trzymaj - Object dobry na wszystko :)